PhysicsTools/PatAlgos/test/miniAOD/patTuple_mini_data.py

```diff
@@ -45,5 +45,3 @@
 #                                         ##
 process.out.fileName = 'patTuple_mini_singlemu.root'
 process.out.outputCommands = process.MicroEventContent.outputCommands
-from PhysicsTools.PatAlgos.slimming.miniAOD_tools import miniAOD_customizeOutput
-miniAOD_customizeOutput(process.out)
```

Comment:
It is unclear to me where the `process.out` comes from, and therefore if the settings from `Configuration/EventContent/python/EventContent_cff.py` would be applied. From the `outputCommands` line above I'd guess they would not be. The question is then if that is important to be kept.

(same for the other test(?) configuration files below)